ractor/Cargo.toml

```diff
@@ -44,20 +47,48 @@ tracing = { version = "0.1", features = ["attributes"] }
 
 ## Blanket Serde
 serde = { version = "1", optional = true }
-pot =  { version = "3.0", optional = true }
+pot = { version = "3.0", optional = true }
+
+
+[target.'cfg(all(target_arch = "wasm32",target_os = "unknown"))'.dependencies]
+tokio_with_wasm = { version = "0.8.2", features = [
+    "macros",
+    "sync",
+    "rt",
+    "time",
+] }
+web-time = "1.1.0"
 
 [dev-dependencies]
 backtrace = "0.3"
-criterion = "0.5"
+
 function_name = "0.3"
 paste = "1"
 serial_test = "3.0.0"
 rand = "0.8"
-tokio = { version = "1.30", features = ["rt", "time", "sync", "macros", "rt-multi-thread", "tracing"] }
 tracing-glog = "0.4"
-tracing-subscriber = { version = "0.3", features = ["env-filter"]}
+tracing-subscriber = { version = "0.3", features = ["env-filter"] }
+
+ractor_example_entry_proc = { version = "0.15.0", path = "../ractor_example_entry_proc" }
```

Comment:
ok I see it now, this doesn't need to be published. In that case make the version 0.0.0 and don't reference the version here. These crate versions don't need to be incremented since they aren't publicly available.

Reply:
Fixed it now